src/chai/ArrayManager.hpp

```diff
@@ -416,6 +416,18 @@ class ArrayManager
    */
   bool syncIfNeeded();
 
+#if defined(CHAI_ENABLE_GPU_SIMULATION_MODE)
+  /*!
+   * \brief Turn the GPU simulation mode on or off.
+   */
+  void setGPUSimMode(bool gpuSimMode) { m_gpu_sim_mode = gpuSimMode; }
+
+  /*!
+   * \brief Return true if GPU simulation mode is on, false otherwise.
+   */
+  bool getGPUSimMode() { return m_gpu_sim_mode; }
```

Comment:
```suggestion
  bool inGpuSimMode() { return m_gpu_sim_mode; }
```

I'd rather this be an "is" or "is" method, I think it's more obvious it's returning a boolean.

Reply:
Fixed.